Florent Chuffart  
INSERM/IAB/EpiMed (U1209/UMR5309)  
Site Santé - Allée des Alpes  
38700 La Tronche  
florent.chuffart@univ-grenoble-alpes.fr  
tel: +33 (0)4 76 54 95 82  
fax: +33 (0)4 76 54 94 25  


# Research Publications

[20]
*Direct visualization of pre-protamine 2 detects protamine assembly failures and predicts ICSI success*
Maryam Rezaei-Gazik, Alexandra Vargas, Amir Amiri-Yekta, Anne-Laure Vitte, Arvand Akbari, Sophie Barral,
Vahid Esmaeili, **Florent Chuffart**, Mohammad Ali Sadighi-Gilani, Yohann Coute, Poopak Eftekhari-Yazdi, Saadi Khochbin, Sophie Rousseaux and Mehdi Totonchi.
**MRH**, 2022.

[19]
*AKR1B10, one of the triggers of cytokine storm in SARS-Cov2 severe Acute Respiratory Syndrome.*
Clovis Chabert, Anne-Laure Vitte, Domenico Iuso, **Florent Chuffart**, 
Candice Trocme, Marlyse Buisson, Pascal Poignard, Benjamin Lardinois, Régis 
Debois, Sophie Rousseaux, Jean-Louis Pepin, Jean-Benoit Martinot, Saadi Khochbin.
**IJMS**, 2022.

[18]
*ATAD2 controls chromatin-bound HIRA turnover.*
Tao Wang, Daniel Perazza, Fayçal Boussouar, Matteo Cattaneo, Alexandre Bougdour, **Florent Chuffart**, Sophie Barral, Alexandra Vargas, Ariadni Liakopoulou, Denis Puthier, Lisa Bargier, Yuichi Morozumi, Mahya Jamshidikia, Isabel Garcia-Saez, Carlo Petosa, Sophie Rousseaux, André Verdel, Saadi Khochbin. 
**Life Sci Alliance**, 2021.

[17]
*CCM2 deficient endothelial cells undergo a mechano-dependent reprogramming into senescence associated secretory phenotype used to recruit endothelial and immune cells.*
Daphné Raphaelle Vannier, Apeksha Shapeti, **Florent Chuffart**, Emmanuelle Plan


[15]
*Chidamide inhibits the NOTCH1-MYC signaling axis in T-cell acute lymphoblastic leukemia.*
Mengping Xi, Shanshan Guo, Caicike Bayin, Lijun peng, **Florent Chuffart**, Ekaterina Bourova-Flin, Sophie Rousseaux, Saadi Khochbin, Jianqing Mi, Jin Wang.
**Frontiers in Medicine**, *under Revision*,

[14]
*The combined detection of Amphiregulin, Cyclin A1 and DDX20/Gemin3 expression predicts aggressive forms of Oral Squamous Cell Carcinoma.*
Ekaterina Bourova-Flin, Samira Derakhshan, Afsaneh Goudarzi, Tao Wang, Anne-Laure Vitte, **Florent Chuffart**, Saadi Khochbin, Sophie Rousseaux, Pouyan Aminishakib. 
**British Journal of Cancer**, *accepted 28th May 2021*,

[13]
*Immediate and durable effects of maternal tobacco consumption alter placental DNA methylation in enhancer and imprinted gene-containing regions.*
Sophie Rousseaux, Emie Seyve, **Florent Chuffart**, Ekaterina Bourova-Flin, Meriem Benmerad, Marie-Aline Charles, Anne Forhan, Barbara Heude, Valérie Siroux, Remy Slama, Jorg Tost, Daniel Vaiman, Saadi Khochbin, Johanna Lepeule, EDEN Mother-Child Cohort Study Group.
**BMC Med**, 2020.


[12]
*PenDA, a Rank-Based Method for Personalized Differential Analysis: Application to Lung Cancer.*
Magali Richard, Clémentine Decamps, **Florent Chuffart**, Elisabeth Brambilla, Sophie Rousseaux, Saadi Khochbin, Daniel Jost.
**PLoS Comput Biol**, 2020.

[11]
*RNA-Guided Genomic Localization of H2A.L.2 Histone Variant.*
Naghmeh Hoghoughi Sophie Barral Sandrine Curtet, **Florent Chuffart**, Guillaume Charbonnier, Denis Puthier, Thierry Buchou, Sophie Rousseaux, Saadi Khochbin.
**Cells**, 2020.

[10]
*Extracellular Vesicles From Myelodysplastic Mesenchymal Stromal Cells Induce DNA Damage and Mutagenesis of Hematopoietic Stem Cells Through miRNA Transfer.* 
Mathieu Meunier, Audrey Guttin, Sarah Ancelet, David Laurin, Johanna Zannoni, Christine Lefebvre, Sylvie Tondeur, Virginie Persoons, Mylène Pezet, Karin Pernet-Gallay, **Florent Chuffart**, Sophie Rousseaux, Quentin Testard, Julien Thevenon, Claire Jouzier, Jean-François Deleuze, Karine Laulagnier, Rémy Sadoul, Christine Chatellard, Pierre Hainaut, Benoît Polack, Jean-Yves Cahn, Jean-Paul Issartel, Sophie Park. 
**Leukemia**, 2020.

[9]
*Guidelines for cell-type heterogeneity quantification based on a comparative analysis of reference-free DNA methylation deconvolution software.*
Clémentine Decamps, Florian Privé, Raphael Bacher, Daniel Jost, Arthur Waguet, **HADACA consortium**, Eugene Andres Houseman, Eugene Lurie, Pavlo Lutsik, Aleksandar Milosavljevic, Michael Scherer, Michael G. B. Blum Magali Richard.
**BMC Bioinformatics**, 2020.

[8]
*Nut Directs p300-Dependent, Genome-Wide H4 Hyperacetylation in Male Germ Cells.*
Shiota H, Barral S, Buchou T, Tan M, Couté Y, Charbonnier G, Reynoird N, Boussouar F, Gérard M, Zhu M, Bargier L, Puthier D, **Chuffart F**, Bourova-Flin E, Picaud S, Filippakopoulos P, Goudarzi A, Ibrahim Z, Panne D, Rousseaux S, Zhao Y, Khochbin S.
**Cell Rep**, 2018.

[7]
*Pregnancy exposure to atmospheric pollution and meteorological conditions and placental DNA methylation.*
Emilie Abraham, Sophie Rousseaux, Lydiane Agier, Lise Giorgis-Allemand, Jörg Tost, Julien Galineau, Agnès Hulin, Valérie Siroux, Daniel Vaiman, Marie-Aline Charles, Barbara Heude, Anne Forhan, Joel Schwartz, **Florent Chuffart**, Ekaterina Bourova-Flin, Saadi Khochbin, Rémy Slama, Johanna Lepeule.
**Environment International**, 2018.

[6]
*Assigning function to natural allelic variation via dynamic modeling of gene network induction.*
Richard M, **Chuffart F**, Duplus-Bottin H, Pouyet F, Spichty M, Fulcrand E, Entrevan M, Barthelaix A, Springer M, Jost D, Yvert G. 
**Mol Syst Biol**, 2018.

[5]
*Exploiting single-cell quantitative data to map genetic variants having probabilistic effects.* 
**Chuffart F**, Richard M, Jost D, Burny C, Duplus-Bottin H, Ohya Y, Yvert G. 
**PLOS Genetics**, 2016.

[4]
*The complex pattern of epigenomic variation between natural yeast strains at single-nucleosome resolution.* 
Fabien Filleton, **Florent Chuffart**, Muniyandi Nagarajan, Helene Bottin-Duplus, Gael Yvert. 
**Epigenetics and Chromatin**, 2015.

[3]
*MyLabStocks: a web-application to manage molecular biology materials.*
**Florent Chuffart** and Gael Yvert. 
**Yeast**, 2014. 

[2]
*When Self-Stabilization Meets Real Platforms: an Experimental Study of a Peer-to-Peer Service Discovery System.*
Eddy Caron, **Florent Chuffart**, Cédric Tedeschi.
**Future Generation Computer Systems**, 2013.

[1]
 *SimExplorer : Programming Experimental Designs on Models and Managing Quality of Modelling Process.*
**Florent Chuffart**, Nicolas Dumoulin, Thierry Faure, Guillaume Deffuant. 
**International Journal of Agricultural and Environmental Information Systems**, 2010.



# Software Production

**dmethr**: Hunting DNA methylation repression of transcriptional activity and its reactivation in a demethylating context https://github.com/fchuffar/dmethr

**medepir**: DNA MEthylation DEconvolution PIpeline in R https://github.com/bcm-uga/medepir

**methepic_map**: An R script that build a dictionary of probes according to array annotations https://github.com/fchuffar/methepic_map

**pipeline_mnase**: A set of scripts that orchestrate the analysis of MNase-Seq and ChIP-Seq data https://github.com/fchuffar/pipeline_mnase

**pipeline_rrbs**: A set of scripts that orchestrate the analysis of RRBS data https://github.com/fchuffar/pipeline_rrbs

**ectopy**: A Python package that detects ectopic gene expression in cancer https://github.com/epimed/ectopy

**ector**: An R package that detects ectopic gene expression in cancer https://github.com/mpetrier/ector

**deeptoolsr**: DeepTools binding for R. https://github.com/fchuffar/deeptoolsr

**penda**: An R package that performs personalized differential analysis of omics data. https://github.com/bcm-uga/penda

**dmprocr**: a set of usefull functions organized around a workflow and performing differential methylation profiles clustering. https://github.com/bcm–uga/dmprocr

**epimedtools**: A package providing a set of useful statistical functions. This package relies on 'GEOquery' and 'affy' packages, R Reference Classes, caching features and memoisation to simplify, homogenize and speed up multi–omic analysis. https://github.com/fchuffar/epimedtools

**methmybeachup**: A package providing a set of functions allowing to easily convolves and visualize the methylome signal across genes and CpG islands. https://github.com/fchuffar/methmybeachup

**BoT**: (stands for Bag of Tasks) is an R package allowing to distribute independent tasks over many cores and many computing nodes. https://github.com/fchuffar/bot

**ptlmapper**: a package which contains functions that are used to map single–cell Probabilistic Trait Loci (scPTL). The principle of scPTL mapping is to consider a distribution of single–cell traits as the phenotype of a (multicellular) individual, to acquire this phenotype in many individuals, and to scan the genome for DNA variants that change this phenotype. https://github.com/fchuffar/ptlmapper




# Training

2021: Atelier Statistique : Traitement statistique de données censurées en analyse de survie, 5, 6, 7 juillet 2021, distanciel.

2020: Single Cell: Génomique de la cellule unique. 8-9 octobre 2020, Lyon (69), France.

2019: Health data challenge (2nd edition), Matrix factorization and deconvolution methods to quantify tumor heterogeneity in cancer. November 29, 2019, Aussois (France), https://cancer-heterogeneity.github.io/data_challenges.html

2018: Health Data Challenge: Matrix factorization and deconvolution methods to quantify tumor heterogeneity in cancer research, 10-14 decembre 2018, Aussois (73), France.

2018 : Biologie intégrative ou biologie des systèmes par Joëlle Henry-Berger,  5-9 novembre 2018, Lyon (69), France.

2018: Journée Campagnes de calculs reproductibles, 23 october, Lyon (69), France.

2017: Epigenetic & High-Dimension Mediation Data Challenge. 7-9 June, Aussois (73), France.

2017: Rencontres scientifiques des Grands Causses : Modélisation physique de l’organisation nucléaire et de ses pathologies. 9-12 Mai, Millau (12), France.

2016: Capture de conformation de chromosomes : vers l’analyse tridimensionnelle de la régulation des génomes. 2-4 Mai, Bordeaux (33), France.

2015: DU in statistics at University of Strasbourg. 19-23 January, 15-19 June, 16-20 November, Strasbourg (67), France.

2015: "R pour le calcul", R avancé et performances. 5-9 October, Aussois (73), France.

2015: Advanced Lecture Course on Computational Systems Biology. 6-11 April, Aussois (73), France.

2012: Advances Lecture Course Hybrid Multicore Parallel Programming, 8-12 October, Autrans (38), France.

2012: Wellcome Trust Advanced Course: In Silico Systems Biology. 23-27 April, Cambridge, UK.

2009: Mexico Advanced Course in Sensitivity Analysis. 11-14 May, Hyères (83), France.




# Jobs

Since December 2015: Research Engineer (IR) at IAB, INSERM, Grenoble, France. Data analysis and modelling of biological systems.

September 2012 - November 2015: Post Doctoral position at LBMC, CNRS, Lyon, France. Dynamic simulation of a gene regulatory network in various wild genetic backgrounds of yeast. project SiGHT (ERC-StG2011-281359) (Systems Genetics of Heritable variaTions)

September 2009 - August 2012: Senior Engineer position at LIP, Inria, Lyon, France. Design and implementation of a peer-to-peer grid middleware supporting petascale architecture. SPADES project (08-ANR-SEGI-025) (Servicing Petascale Architectures and DistributEd System)

Janury 2008 - August 2009: Post Doctoral position at Irstea (ex-Cemagref), Clermont-Ferrand, France. Compared study, design and implementation of Individual Based Model (IBM) and Differential Equation Model (DEM). Application to predator-prey model that incorporates individual behavior of the predators. Sensitivity analysis of from model parameters, impact on model behavior. LifeGrid / PatRes project (FP6-043268) (Pattern Resilience)

October 2004 - December 2007: Ph.D. position at France Telecom / University of Caen Basse-Normandie, Caen, France. Conception design and implementation of an ubiquitous services platform integrating distributed multimodal interfaces. AURORA Project (ITEA-03005)

# Education

2016: DU degree in statistics at University of Strasbourg,Strasbourg, France.

2007: Ph.D. degree in computer science at University of Caen Basse-Normandie, Caen, France.

2004: DEA (master science) in computer science, artificial intelligence and algorithm at University of Caen Basse-Normandie, Caen, France.

2003: Maitrise (master’s degree) in applied Mathematics at University of Rouen, Rouen, France.

2002: Licence (under graduate) in Mathematics at University of Rouen, Rouen, France.

1999: DUT (two-year university degree in technology) in Chemistry at IUT (University Institute of Technology) of Rouen, Rouen, France.

1997: Baccalauréat (French secondary school diploma), science major, Rouen, France.

# Other Communications

*Lysine Methyltransferases Signaling: Histones are Just the Tip of the Iceberg.*
Lukinović V, Casanova AG, Roth GS, **Chuffart F**, Reynoird N.
**Curr Protein Pept Sci.** 2020 Jan 1

*Analysis of RNA-seq data.* 
Florent Chuffart. 
**Journée thematique ’statistique & genomique’**, 19 september 2019, Paris.

*Snakemake as workflow engine*. 
Florent Chuffart. 
**Python working session**. 20 june 2019, Grenoble.

*Histone variants: critical determinants in tumour heterogeneity.* 
Wang T, **Chuffart F**, Bourova-Flin E, Wang J, Mi J, Rousseaux S, Khochbin S. 
**Front Med** 13 (3), 289-297.  Jun 2019.

*ROCKs dependent contractility upon CCM depletion leads to a lossof endothelial cell identity and a gain in invasiv phenotype.* 
Daphné Vannier et al. Poster. 
**Developmental and cell biology of the future conference**, 27-29 march 2019, Paris.

Utilisation des ressources CIMENT dans le cadre du projet e*pimed: les besoins spécifiques de la bioinformatique pour l’analyse des données d’exome.* 
Quentin Testard, Julien Thevenon, Jean-François Taly, Laure Raymond, **Florent Chuffart.** 
**JCAD** 2018, 24-26 october, Lyon (69), France.

*Intégration de données multi-omiques: RGCCA, Mix Omics.* 
Florent Chuffart. 
**Journée thematique ’statistique & genomique’**, 12 octobre 2018, Paris.


*Retour d’expérience concernant le data challenge épigénétique et médiation à large échelle d’Aussois.*
Sophie Achard, Dylan Aïssi, Michael Blum, Kevin Caye, **Florent Chuffart**, Olivier François, Keurcien Luu, Florian Privé, Magali Richard.
**CFIES2017**, septembre 2017, Grenoble, France. https://toltex.imag.fr/users/RCqls/Workshop/cfies2017/Submissions/subm44.pdf

*The role of bromodomain testis-specific factor, BRDT, in cancer: a biomarker and a possible therapeutic target.* 
Ekaterina Flin, **Florent Chuffart**, Sophie Rousseaux, Saadi Khochbin. 
**Cell J** (Yakhteh) Volume 19, Supplement 1, Spring 2017

*Heterogenety in Carcinogenesis Mechanism (the lung cancer example)* 
Florent Chuffart. 
**Kick-off PITCHER**. Lyon, 9 mach 2017.

*execo* (APP creation deposit). Matthieu Imbert, Laurent Pouilloux, **Florent Chuffart.** APP. 2015.

*A General Method to map Single-cell Probabilistic Trait Loci of the Genome.* (Short Talk). 
Florent Chuffart.
**Quatrièmes Rencontres R**, 24-26 June 2015, Grenoble, France.

*Functional Genetic Diversity of the Yeast Galactose Network.* (Short Talk). 
Florent Chuffart.
**CompSysBio**, 6-11 April 2015, Aussois, France.

*NucleoMiner 2.0: Detecting intra-species quantitative epigenomic variation at single-nucleosome resolution.*
**Florent Chuffart**, Gael Yvert. (Poster). 
**LyonSysBio**, 19-21 November 2014, Villeurbanne, France.

*Parallel Computing in R using the Bot Package.* 
Florent Chuffart (Poster). 
**Deuxièmes rencontres R**. 27-28 Juin 2013, Lyon, France.

*Optimization in a Self-stabilizing Service Discovery Framework for Large Scale Systems.* (in Proceedings). 
Eddy Caron, **Florent Chuffart**, Anissa Lamani, Franck Petit.
**SSS**. 2012. 239-252.

*Large scale P2P discovery middleware demonstration.* (in Proceedings). Eddy Caron, **Florent Chuffart**, Haiwu He, Anissa Lamani, Philippe Le Brouster, Olivier Richard.
**IEEE International Conference on Peer-to-Peer Computing** (P2P), 1-2 September 2011, Kyoto, Japan. 152-153

*Mise en oeuvre de la découverte de services pour des plateformes dynamiques à large échelle.* (in Proceedings). 
**Florent Chuffart**, Haiwu He.
**RenPar’20**, 10-13 May 2011, Saint-Malo, France. RSTI, serie TSI, 31:8-9-10, 1101-1119.

*Implementation and Evaluation of a P2P Service Discovery System: Application in a Dynamic Large Scale Computing Infrastructure.* (in Proceedings). 
Caron, E.; **Chuffart, F.**; Haiwu He; Tedeschi, C. 
**IEEE 11th International Conference on Computer and Information Technology** (CIT). 2011. 41-46.

*Declarative task delegation in OpenMOLE.* (in Proceedings). 
R Reuillon, **F Chuffart**, M Leclaire, T Faure, N Dumoulin, D Hill. 
**International Conference on High performance computing and simulation** (HPCS). 2010. 55-62.

*Applying a Moment Approximation to a bacterial biofilm individual-based model.* (in Proceedings). 
J-D. Mathias, **F. Chuffart**, N. Mabrouk, and G. Deffuant. 
**ICCGI 2009** : Proceedings of the 2009 Fourth International Multi-Conference on Computing in the Global Information Technology, pages 138-143, Washington, DC, USA, 2009. IEEE Computer Society.

*Record & replay: expérimentation d’un service multimodal.* (in Proceedings). 
Antoine Bouyer, **Florent Chuffart**.
**UbiMob ’09** Proceedings of the 5th French-Speaking Conference on Mobility and Ubiquity Computing. 2009. 19-23.

*The Design of a Multimodal Platform: Experimentation of Record & Replay.* (in Proceedings). 
Bouyer, A. ; **Chuffart, F.** ; Courval, L. 
**Second International Conferences on Advances in Computer-Human Interactions**. 2009. 1-6.

*Design and Analysis of Computer Experiments with SimExplorer.* (in Proceedings). 
**Florent Chuffart**, Thierry Faure, Nicolas Dumoulin, Romain Reuillon, and Guillaume Deffuant. 
**European Simulation and Modelling Conference** (ESM2008). 27-29 October 2008, Le Havre, France. 575-577.

*Conception d’une plate-forme de services ubiquitaires intégrant des interfaces multimodales distribuées.* (PhD thesis). 
Florent Chuffart. Université de Caen Basse-Normandie. 2007.

*Construction d’une application vocale pour la sélection d’objets à l’aide d’un modèle basé sur les hypergraphes.* (in Proceedings). 
Cyril Bazin, **Florent Chuffart**, and Jacques Madelaine. 
**9ème Conférence Internationale sur le Document Numérique**. 18-20 Septembre 2006, Fribourg, Suisse. 43-58.

*Aurora, multimodal messaging framework for ubiquitous web context.* (Communication). 
**F. Chuffart**, F. Van Gool, and L. Courval. 
**W3C Ubiquitous Web Workshop**, 2006.

*Aurora, a framework enabling multimodal interactions.* (in Proceedings). 
**Florent Chuffart**, Filip Van Gool, and Lionel Courval. 
In Workshop on International on Multimodal Multiparty Meeting Processing, ACM **ICMI Workshop**. 7 October 2005, Trento, Italy. 53-585.

