<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>index.utf8.md</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Florent Chuffart</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class="fa fa-user"></span>
     
    About
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:florent.chuffart@gmail.com">
    <span class="fa fa-envelope-o"></span>
     
    Contact me
  </a>
</li>
<li>
  <a href="https://github.com/fchuffar">
    <span class="fa fa-github"></span>
     
    GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p>INSERM/IAB/EpiMed (U1209/UMR5309)<br />
Site Santé - Allée des Alpes<br />
38700 La Tronche<br />
<a href="mailto:florent.chuffart@univ-grenoble-alpes.fr" class="email">florent.chuffart@univ-grenoble-alpes.fr</a><br />
tel: +33 (0)4 76 54 95 82<br />
fax: +33 (0)4 76 54 94 25</p>
<div id="jobs" class="section level1">
<h1>Jobs</h1>
<p>Since December 2015: Research Engineer (IR) at IAB, INSERM, Grenoble, France. Data analysis and modelling of biological systems.</p>
<p>September 2012 - November 2015: Post Doctoral position at LBMC, CNRS, Lyon, France. Dynamic simulation of a gene regulatory network in various wild genetic backgrounds of yeast. project SiGHT (ERC-StG2011-281359) (Systems Genetics of Heritable variaTions)</p>
<p>September 2009 - August 2012: Senior Engineer position at LIP, Inria, Lyon, France. Design and implementation of a peer-to-peer grid middleware supporting petascale architecture. SPADES project (08-ANR-SEGI-025) (Servicing Petascale Architectures and DistributEd System)</p>
<p>Janury 2008 - August 2009: Post Doctoral position at Irstea (ex-Cemagref), Clermont-Ferrand, France. Compared study, design and implementation of Individual Based Model (IBM) and Differential Equation Model (DEM). Application to predator-prey model that incorporates individual behavior of the predators. Sensitivity analysis of from model parameters, impact on model behavior. LifeGrid / PatRes project (FP6-043268) (Pattern Resilience)</p>
<p>October 2004 - December 2007: Ph.D. position at France Telecom / University of Caen Basse-Normandie, Caen, France. Conception design and implementation of an ubiquitous services platform integrating distributed multimodal interfaces. AURORA Project (ITEA-03005)</p>
</div>
<div id="training" class="section level1">
<h1>Training</h1>
<p>2019: Health data challenge (2nd edition), Matrix factorization and deconvolution methods to quantify tumor heterogeneity in cancer. November 29, 2019, Aussois (France), <a href="https://cancer-heterogeneity.github.io/data_challenges.html" class="uri">https://cancer-heterogeneity.github.io/data_challenges.html</a></p>
<p>2018: Health Data Challenge: Matrix factorization and deconvolution methods to quantify tumor heterogeneity in cancer research, 10-14 decembre 2018, Aussois (73), France.</p>
<p>2018 : Biologie intégrative ou biologie des systèmes par Joëlle Henry-Berger, 5-9 novembre 2018, Lyon (69), France.</p>
<p>2018: Journée Campagnes de calculs reproductibles, 23 october, Lyon (69), France.</p>
<p>2017: Epigenetic &amp; High-Dimension Mediation Data Challenge. 7-9 June, Aussois (73), France.</p>
<p>2017: Rencontres scientifiques des Grands Causses : Modélisation physique de l’organisation nucléaire et de ses pathologies. 9-12 Mai, Millau (12), France.</p>
<p>2016: Capture de conformation de chromosomes : vers l’analyse tridimensionnelle de la régulation des génomes. 2-4 Mai, Bordeaux (33), France.</p>
<p>2015: DU in statistics (3/3) at University of Strasbourg. 16-20 November, Strasbourg (67), France.</p>
<p>2015: “R pour le calcul”, R avancé et performances. 5-9 October, Aussois (73), France.</p>
<p>2015: DU in statistics (2/3) at University of Strasbourg. 15-19 June Strasbourg (67), France.</p>
<p>2015: Advanced Lecture Course on Computational Systems Biology. 6-11 April, Aussois (73), France.</p>
<p>2015: DU in statistics (1/3) at University of Strasbourg. 19-23 January, Strasbourg (67), France.</p>
<p>2012: Advances Lecture Course Hybrid Multicore Parallel Programming, 8-12 October, Autrans (38), France.</p>
<p>2012: Wellcome Trust Advanced Course: In Silico Systems Biology. 23-27 April, Cambridge, UK.</p>
<p>2009: Mexico Advanced Course in Sensitivity Analysis. 11-14 May, Hyères (83), France.</p>
</div>
<div id="education" class="section level1">
<h1>Education</h1>
<p>2016: DU degree in statistics at University of Strasbourg,Strasbourg, France.</p>
<p>2007: Ph.D. degree in computer science at University of Caen Basse-Normandie, Caen, France.</p>
<p>2004: DEA (master science) in computer science, artificial intelligence and algorithm at University of Caen Basse-Normandie, Caen, France.</p>
<p>2003: Maitrise (master’s degree) in applied Mathematics at University of Rouen, Rouen, France.</p>
<p>2002: Licence (under graduate) in Mathematics at University of Rouen, Rouen, France.</p>
<p>1999: DUT (two-year university degree in technology) in Chemistry at IUT (University Institute of Technology) of Rouen, Rouen, France.</p>
<p>1997: Baccalauréat (French secondary school diploma), science major, Rouen, France.</p>
</div>
<div id="research-publications" class="section level1">
<h1>Research Publications</h1>
<p>Extracellular Vesicles From Myelodysplastic Mesenchymal Stromal Cells Induce DNA Damage and Mutagenesis of Hematopoietic Stem Cells Through miRNA Transfer. Mathieu Meunier, Audrey Guttin, Sarah Ancelet, David Laurin, Johanna Zannoni, Christine Lefebvre, Sylvie Tondeur, Virginie Persoons, Mylène Pezet, Karin Pernet-Gallay, Florent Chuffart, Sophie Rousseaux, Quentin Testard, Julien Thevenon, Claire Jouzier, Jean-François Deleuze, Karine Laulagnier, Rémy Sadoul, Christine Chatellard, Pierre Hainaut, Benoît Polack, Jean-Yves Cahn, Jean-Paul Issartel, Sophie Park. Leukemia. 2020</p>
<p>Nut Directs p300-Dependent, Genome-Wide H4 Hyperacetylation in Male Germ Cells. Shiota H, Barral S, Buchou T, Tan M, Couté Y, Charbonnier G, Reynoird N, Boussouar F, Gérard M, Zhu M, Bargier L, Puthier D, Chuffart F, Bourova-Flin E, Picaud S, Filippakopoulos P, Goudarzi A, Ibrahim Z, Panne D, Rousseaux S, Zhao Y, Khochbin S. Cell Rep. 2018 Sep 25;24(13):3477-3487.e6.</p>
<p>Pregnancy exposure to atmospheric pollution and meteorological conditions and placental DNA methylation. Emilie Abraham, Sophie Rousseaux, Lydiane Agier, Lise Giorgis-Allemand, Jörg Tost, Julien Galineau, Agnès Hulin, Valérie Siroux, Daniel Vaiman, Marie-Aline Charles, Barbara Heude, Anne Forhan, Joel Schwartz, Florent Chuffart, Ekaterina Bourova-Flin, Saadi Khochbin, Rémy Slama, Johanna Lepeule. Environment International. 2018. ISSN 0160-4120.</p>
<p>Assigning function to natural allelic variation via dynamic modeling of gene network induction. Richard M, Chuffart F, Duplus-Bottin H, Pouyet F, Spichty M, Fulcrand E, Entrevan M, Barthelaix A, Springer M, Jost D, Yvert G. Mol Syst Biol. 2018. Jan 15;14(1):e7803.</p>
<p>Exploiting single-cell quantitative data to map genetic variants having probabilistic effects. Chuffart F, Richard M, Jost D, Burny C, Duplus-Bottin H, Ohya Y and Yvert G. PLOS Genetics. 2016. Volume 12, Issue 8, pages 1-27.</p>
<p>The complex pattern of epigenomic variation between natural yeast strains at single-nucleosome resolution. Fabien Filleton, Florent Chuffart, Muniyandi Nagarajan, Helene Bottin-Duplus, Gael Yvert. Epigenetics &amp; Chromatin. 2015. Jul 31;8:26.</p>
<p>MyLabStocks: a web-application to manage molecular biology materials. Florent Chuffart and Gael Yvert. Yeast. 2014. Volume 31, Issue 5, pages 179-184.</p>
<p>When Self-Stabilization Meets Real Platforms: an Experimental Study of a Peer-to-Peer Service Discovery System. Eddy Caron, Florent Chuffart, Cédric Tedeschi Future Generation Computer Systems. 2013. 29, 6 1533-1543.</p>
<p>SimExplorer : Programming Experimental Designs on Models and Managing Quality of Modelling Process. Florent Chuffart, Nicolas Dumoulin, Thierry Faure, and Guillaume Deffuant. International Journal of Agricultural and Environmental Information Systems. 2010. 1(1) :55-68.</p>
</div>
<div id="other-communications" class="section level1">
<h1>Other Communications</h1>
<p>Lysine Methyltransferases Signaling: Histones are Just the Tip of the Iceberg. Lukinović V, Casanova AG, Roth GS, Chuffart F, Reynoird N. Curr Protein Pept Sci. 2020 Jan 1</p>
<p>Maternal exposure to cigarette smoking induces immediate and durable changes in placental DNA methylation affecting enhancer and imprinting control regions Sophie Rousseaux, Emie Seyve, Florent Chuffart, Ekaterina Bourova-Flin, MeriemBenmerad, Marie-Aline Charles, Anne Forhan, Barbara Heude, Valérie Siroux, RémySlama, Jorg Tost, Daniel Vaiman, Saadi Khochbin, Johanna Lepeule. bioRxiv 852186; doi: <a href="https://doi.org/10.1101/852186" class="uri">https://doi.org/10.1101/852186</a></p>
<p>Analysis of RNA-seq data. Florent Chuffart. Journée thematique ’statistique &amp; genomique’, 19 september 2019, Paris.</p>
<p>Snakemake as workflow engine. Florent Chuffart. Python working session. 20 june 2019, Grenoble.</p>
<p>Histone variants: critical determinants in tumour heterogeneity. Wang T, Chuffart F, Bourova-Flin E, Wang J, Mi J, Rousseaux S, Khochbin S. Front Med 13 (3), 289-297. Jun 2019.</p>
<p>ROCKs dependent contractility upon CCM depletion leads to a lossof endothelial cell identity and a gain in invasiv phenotype. Daphné Vannier et al. Poster. Developmental and cell biology of the future conference, 27-29 march 2019, Paris.</p>
<p>Utilisation des ressources CIMENT dans le cadre du projet epimed: les besoins spécifiques de la bioinformatique pour l’analyse des données d’exome. Quentin Testard, Julien Thevenon, Jean-François Taly, Laure Raymond, Florent Chuffart. JCAD 2018, 24-26 october, Lyon (69), France.</p>
<p>Intégration de données multi-omiques: RGCCA, Mix Omics. Florent Chuffart. Journée thematique ’statistique &amp; genomique’, 12 octobre 2018, Paris.</p>
<p><em>Retour d’expérience concernant le data challenge épigénétique et médiation à large échelle d’Aussois.</em> Sophie Achard, Dylan Aïssi, Michael Blum, Kevin Caye, Florent Chuffart, Olivier François, Keurcien Luu, Florian Privé, Magali Richard. <strong>CFIES2017</strong>, septembre 2017, Grenoble, France. <a href="https://toltex.imag.fr/users/RCqls/Workshop/cfies2017/Submissions/subm44.pdf" class="uri">https://toltex.imag.fr/users/RCqls/Workshop/cfies2017/Submissions/subm44.pdf</a></p>
<p>The role of bromodomain testis-specific factor, BRDT, in cancer: a biomarker and a possible therapeutic target. Ekaterina Flin, Florent Chuffart, Sophie Rousseaux, Saadi Khochbin. Cell J (Yakhteh) Volume 19, Supplement 1, Spring 2017</p>
<p>Heterogenety in Carcinogenesis Mechanism (the lung cancer example) Florent Chuffart. Kick-off PITCHER. Lyon, 9 mach 2017.</p>
<p>execo (APP creation deposit). Matthieu Imbert, Laurent Pouilloux, Florent Chuffart. APP. 2015.</p>
<p>A General Method to map Single-cell Probabilistic Trait Loci of the Genome. (Short Talk). Quatrièmes Rencontres R, 24-26 June 2015, Grenoble, France.</p>
<p>Functional Genetic Diversity of the Yeast Galactose Network. (Short Talk). CompSysBio, 6-11 April 2015, Aussois, France.</p>
<p>NucleoMiner 2.0: Detecting intra-species quantitative epigenomic variation at single-nucleosome resolution. (Poster). LyonSysBio, 19-21 November 2014, Villeurbanne, France.</p>
<p>Parallel Computing in R using the Bot Package. (Poster). Deuxièmes rencontres R. 27-28 Juin 2013, Lyon, France.</p>
<p>Optimization in a Self-stabilizing Service Discovery Framework for Large Scale Systems. (in Proceedings). Eddy Caron, Florent Chuffart, Anissa Lamani, Franck Petit SSS. 2012. 239-252.</p>
<p>Large scale P2P discovery middleware demonstration. (in Proceedings). Eddy Caron, Florent Chuffart, Haiwu He, Anissa Lamani, Philippe Le Brouster, Olivier Richard IEEE International Conference on Peer-to-Peer Computing (P2P), 1-2 September 2011, Kyoto, Japan. 152-153</p>
<p>Mise en oeuvre de la découverte de services pour des plateformes dynamiques à large échelle. (in Proceedings). Florent Chuffart, Haiwu He RenPar’20, 10-13 May 2011, Saint-Malo, France. RSTI, serie TSI, 31:8-9-10, 1101-1119.</p>
<p>Implementation and Evaluation of a P2P Service Discovery System: Application in a Dynamic Large Scale Computing Infrastructure. (in Proceedings). Caron, E.; Chuffart, F.; Haiwu He; Tedeschi, C. IEEE 11th International Conference on Computer and Information Technology (CIT). 2011. 41-46.</p>
<p>Declarative task delegation in OpenMOLE. (in Proceedings). R Reuillon, F Chuffart, M Leclaire, T Faure, N Dumoulin, D Hill. International Conference on High performance computing and simulation (HPCS). 2010. 55-62.</p>
<p>Applying a Moment Approximation to a bacterial biofilm individual-based model. (in Proceedings). J-D. Mathias, F. Chuffart, N. Mabrouk, and G. Deffuant. In ICCGI ’09 : Proceedings of the 2009 Fourth International Multi-Conference on Computing in the Global Information Technology, pages 138-143, Washington, DC, USA, 2009. IEEE Computer Society.</p>
<p>Record &amp; replay: expérimentation d’un service multimodal. (in Proceedings). Antoine Bouyer, Florent Chuffart UbiMob ’09 Proceedings of the 5th French-Speaking Conference on Mobility and Ubiquity Computing. 2009. 19-23.</p>
<p>The Design of a Multimodal Platform: Experimentation of Record &amp; Replay. (in Proceedings). Bouyer, A. ; Chuffart, F. ; Courval, L. Second International Conferences on Advances in Computer-Human Interactions. 2009. 1-6.</p>
<p>Design and Analysis of Computer Experiments with SimExplorer. (in Proceedings). Florent Chuffart, Thierry Faure, Nicolas Dumoulin, Romain Reuillon, and Guillaume Deffuant. European Simulation and Modelling Conference (ESM2008). 27-29 October 2008, Le Havre, France. 575-577.</p>
<p>Conception d’une plate-forme de services ubiquitaires intégrant des interfaces multimodales distribuées. (PhD thesis). Florent Chuffart. Université de Caen Basse-Normandie. 2007.</p>
<p>Construction d’une application vocale pour la sélection d’objets à l’aide d’un modèle basé sur les hypergraphes. (in Proceedings). Cyril Bazin, Florent Chuffart, and Jacques Madelaine. 9ème Conférence Internationale sur le Document Numérique. 18-20 Septembre 2006, Fribourg, Suisse. 43-58.</p>
<p>Aurora, multimodal messaging framework for ubiquitous web context. (Communication). F. Chuffart, F. Van Gool, and L. Courval. In W3C Ubiquitous Web Workshop. 2006.</p>
<p>Aurora, a framework enabling multimodal interactions. (in Proceedings). Florent Chuffart, Filip Van Gool, and Lionel Courval. In Workshop on International on Multimodal Multiparty Meeting Processing, ACM ICMI Workshop. 7 October 2005, Trento, Italy. 53-585.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
